api/repositories/org_repository.go

```diff
@@ -125,61 +129,105 @@ func (r *OrgRepo) CreateOrg(ctx context.Context, info authorization.Info, org Cr
 	if err != nil {
 		return OrgRecord{}, fmt.Errorf("failed to build user client: %w", err)
 	}
-	orgGUID := OrgPrefix + uuid.NewString()
-	anchor, err := r.createSubnamespaceAnchor(
-		ctx,
-		info,
-		userClient,
-		&v1alpha2.SubnamespaceAnchor{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      orgGUID,
-				Namespace: r.rootNamespace,
-				Labels: map[string]string{
-					OrgNameLabel: org.Name,
-				},
-			},
+	var orgCR *workloads.CFOrg
+	orgCR, err = r.createOrgCR(ctx, info, userClient, &workloads.CFOrg{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      OrgPrefix + uuid.NewString(),
+			Namespace: r.rootNamespace,
 		},
-		OrgResourceType,
-	)
+		Spec: workloads.CFOrgSpec{
+			Name: org.Name,
+		},
+	})
+
 	if err != nil {
-		if webhookError, ok := webhooks.WebhookErrorToValidationError(err); ok {
+		if webhookError, ok := webhooks.WebhookErrorToValidationError(err); ok { // untested
```

Comment:
I have always wondered what is the purpose of those `//untested` comments. Isn't codeclimate's report better to track such sort of things? Or, if you think that testing this branch is important but is hard, maybe filing a chore about it would speed up fixing that.

Reply:
I usually add these comments to make it clear that we chose not to test that line, rather than it being an oversight. If it's behavior that I'm concerned about testing, then I usually try to find a way to test it instead of leaving the comment.